db/seeds.rb

```diff
@@ -24,7 +24,32 @@
 	description: "my description of Futurama",
 	small_cover_url: "/tmp/futurama.jpg",
 	large_cover_url: "/tmp/futurama.jpg",
-	category_id: 3)
+	category_id: 2)
```

Comment:
Instead of assigning category ids, you should assign the categories to variables. Then you can do:

```ruby
category: comedy
```

instead of:

```ruby
category_id: 2
```